docs/NotebookInstructions.md

```diff
@@ -14,29 +14,16 @@ This example notebook demonstrates how to use the [Dataset Insights](https://git
 * Create a virtual environment named `synthdet_env` using Conda, and activate it:
 
 ```bash
-conda create -n synthdet_env python=3.7
+conda create -n synthdet_env python=3.8
 conda activate synthdet_env
 ```
 
-* Install Dataset Insights:
+* Install datasetinsights package from PyPI:
 
 ```bash
 pip install datasetinsights
 ```
 
-* Install Jupyter:
```

Comment:
Have you checked whether users can run it on both OSX and Windows followed by the new instructions?

Reply:
It works on OSX and I need to double check on Windows VM. I am pretty certain that we don't need to install jupyter as that was already included when you `pip install datasetinsights`

Reply:
I haven't gotten around to it yet.